### PR TITLE
[MIG] product_tax_multicompany_default: Migration to 12.0

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa TT20737

### CODE REVIEW

File: product_tax_multicompany_default/README.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
```
Comment (pedrobaeza) [inline product_tax_multicompany_default/README.rst]: Split the README into parts and generate the new template

File: product_tax_multicompany_default/README.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+This module sets the default company taxes for all the existing companies when
+a product is created. It also adds a button to set all the taxes from other
+companies matching them by tax code.
+
+Configuration
+=============
+
+#. As obvious, you need to have several companies.
+#. Go to Accounting > Configuration > Settings
+#. Select the proper company (if you are admin or a user with several companies
```
Comment (pedrobaeza) [inline product_tax_multicompany_default/README.rst]: In v12, there's no admin that sees everything.

File: product_tax_multicompany_default/README.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+This module sets the default company taxes for all the existing companies when
+a product is created. It also adds a button to set all the taxes from other
+companies matching them by tax code.
+
+Configuration
+=============
+
+#. As obvious, you need to have several companies.
+#. Go to Accounting > Configuration > Settings
```
Comment (pedrobaeza) [inline product_tax_multicompany_default/README.rst]:
```suggestion
#. Go to Invoicing > Configuration > Settings
```

File: product_tax_multicompany_default/README.rst [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+#. Select the proper company (if you are admin or a user with several companies
+   access), or keep the current one (for a regular user).
+#. On Invoicing & Payments section, set fields "Default Sale Tax" and "Default
```
Comment (pedrobaeza) [inline product_tax_multicompany_default/README.rst]:
```suggestion
#. On *Taxes > Default Taxes* section, set fields "Sales Tax" and "
```

File: product_tax_multicompany_default/README.rst [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+   Purchase Tax".
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Sales > Sales > Products.
+#. Create a new product and save it.
+#. Switching user company (or being as admin or user with several companies
```
Comment (pedrobaeza) [inline product_tax_multicompany_default/README.rst]: No admin in v12

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
There's no mention to "Propagate taxes" button in the Usage section.

**Comment by carlosdauden** [comment]
Changes done

File: product_tax_multicompany_default/README.rst [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+* Go to Accounting > Configuration > Settings
+* Select the proper company (if you are admin or a user with several companies
+  access), or keep the current one (for a regular user).
+* On Invoicing & Payments section, set fields "Default Sale Tax" and "Default
+  Purchase Tax".
+
+Usage
+=====
+
+* Go to Sales > Sales > Products.
```
Comment (pedrobaeza) [inline product_tax_multicompany_default/README.rst]: Why have you changed the numbered list by an unordered one?

**Review by pedrobaeza — APPROVED** [review]

File: product_tax_multicompany_default/README.rst (hunk 6 above)
Comment (carlosdauden) [reply product_tax_multicompany_default/README.rst]: Because taking a look I saw that fashion was to put asterisks :smile:

Comment (pedrobaeza) [reply product_tax_multicompany_default/README.rst]: Well, that's old fashion :stuck_out_tongue_closed_eyes: See template when you have doubts: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/readme/USAGE.rst

Comment (carlosdauden) [reply product_tax_multicompany_default/README.rst]: Changes done.
I like to be always up to date. :smiley:

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**